scripts/toolchain.lua

```diff
@@ -253,6 +267,18 @@ function toolchain(_buildDir, _libDir)
 			premake.gcc.ar  = "ar"
 			location (path.join(_buildDir, "projects", _ACTION .. "-" .. _OPTIONS["gcc"]))
 
+		elseif "xros-arm64" == _OPTIONS["gcc"] then
+			premake.gcc.cc  = "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang"
```

Comment:
Don't see difference with xros-simulator, should be `or "xors-simulator"` here instead of whole block?

Reply:
Thanks, applied your suggestion